src/cloudflare/internal/entrypoints.d.ts

```diff
@@ -0,0 +1,7 @@
+export class DurableObject {
+  public constructor(state: unknown, env: unknown);
+}
+
+export class StatelessService {
+  public constructor(ctx: unknown, env: unknown);
```

Comment:
The order of these arguments makes me deeply uncomfortable, but I see why you've done it.

Btw I don't think this is doing what you expect, TS is weird around asserting constructor arguments. However we might want to specify that both of these attach properties to their instance, if that's what they're going to do (otherwise why are they passed to super?)

Reply:
Yes, the ordering is awkward but it seems ore important to be consistent with `DurableObject` than to be consistent with the freestanding functions.

The superclass constructor requires these arguments (despite doing nothing with them) for a couple reasons:
* If the subclass doesn't declare a constructor, the superclass constructor will receive these params by defalut, so it seems like the right thing to do.
* I want to reserve the right to do something with them in the future even if we don't today.